python/dgl/backend/__init__.py

```diff
@@ -8,6 +8,8 @@
 from . import backend
 from .set_default_backend import set_default_backend
 
+from .._ffi.base import set_backend
```

Comment:
Seems that this statement alone causes the crash, even if I add it to master branch code.